src/rc_allocator.h

```diff
@@ -88,6 +88,20 @@ typedef struct rc_allocator {
 } rc_allocator;
 
 
+/* Is page address 'addr' is a valid extent address */
+static inline bool
+rc_allocator_valid_extent_addr(rc_allocator *al, uint64 addr)
+{
+   return (addr % al->cfg->io_cfg->extent_size == 0);
```

Comment:
I thought I would add some validation under the assertion to check if the extent-addr is valid [etc.], but I didn't find this necessary, after all.

Having modularized this to a re-usable function, I left it in for now. If in future, we need external modules to check if an extent-addr is valid, this fn will come in handy. (We'd have to slap a virtual fn interface on top of it.)

Would like to leave this in, even though it's slightly unrelated to this fix, per se'.

Reply:
Also, just so you know, I have retained this sibling method, too, `rc_allocator_extent_num()`.

I want to retain both as they will be useful to call from the debugger, while examining large 'addr' values, to figure out the semantics that these two fns offer.